src/examples/tests.ts

```diff
@@ -3,12 +3,14 @@ import { start, stop } from '../utils/server';
 import * as data from '../utils/elementData';
 import { Plugin } from '../modules/plugin';
 import * as assert from 'assert';
+import { sleep } from '../../dist/src/utils/sleep';
```

Comment:
you imported from dist here, was that intentional?

Reply:
Yep, not really sure what happened here 😬 . Updated to `../utils/sleep'